ql/termstructures/yield/ultimateforwardtermstructure.hpp

```diff
@@ -73,7 +77,10 @@ namespace QuantLib {
                                      Handle<Quote> lastLiquidForwardRate,
                                      Handle<Quote> ultimateForwardRate,
                                      const Period& firstSmoothingPoint,
-                                     Real alpha);
+                                     Real alpha,
+                                     const ext::optional<Integer>& rounding = ext::nullopt,
```

Comment:
Maybe a more descriptive name like `roundingDigits`?

On that subject, we should document if the rounding digits should refer to the rate written, say, as 0.05 or 5%.  I'm guessing the rules probably mention the latter, but we're using the former here.